### PR TITLE
New products popular sections

### PR DESCRIPTION
**Pull request by bynikitachel**
Tasks: 
E-1 S-5 T-1, 
E-1 S-5 T-2,
E-1 S-5 T-3, 
E-1 S-6 T-1, 
E-1 S-6 T-2,
E-1 S-6 T-3, 
E-1 S-6 T-4, 
E-1 S-6 T-5,
E-1 S-7 T-1, 
E-1 S-7 T-2, 
E-1 S-7 T-3, 
E-1 S-7 T-4

### CODE REVIEW

**Comment by imma-fish-u** [comment]
The project isn't running cuz there's no folder stuff, is it mockData instead? Btw it's better to move img and data in folder "assets"

File: src/components/Button/index.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    background: none;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+
+    span {
+        margin: 0 16px;
+    }
+
+    &_primary {
```
Comment (imma-fish-u) [inline src/components/Button/index.scss]: Hey! We use '--' this sign for modificators, it was mentioned in link with Bem info

File: src/components/CarouselItem/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const CarouselItem = ({ currentIndex, length }) => {
+  return (
+    <div className="index-slide">
+      <div className="index-slide__wrapper">
+        <div className="index-slide__wrapper__current-slide">0{currentIndex}/</div>
```
Comment (imma-fish-u) [inline src/components/CarouselItem/index.js]: In BEM is not allowed two '__' signs in 1 style. It's a plain model, so it should be something like "index-slide__current-slide"

File: src/components/Gallery/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import HeaderSection from '../HeaderSection'
+import Button from '../Button/index'
+import Card from '../Card'
+
+const Gallery = ({ items, headerText }) => {
+  return (
+    <section className="gallery">
+      <div className="gallery__container">
+        <div className="gallery__container__header">
```
Comment (imma-fish-u) [inline src/components/Gallery/index.js]: same thing with '__'

**Review by imma-fish-u — CHANGES_REQUESTED** [review]

**Comment by imma-fish-u** [comment]
pls check everything on things like this "gallery__container__header"

File: src/components/Button/index.scss (hunk 1 above)
Comment (bynikitachel) [reply src/components/Button/index.scss]: ok

File: src/components/CarouselItem/index.js (hunk 2 above)
Comment (bynikitachel) [reply src/components/CarouselItem/index.js]: ok

File: src/components/Gallery/index.js (hunk 3 above)
Comment (bynikitachel) [reply src/components/Gallery/index.js]: ok

File: src/pages/home/sections/OurWork/index.js [hunk 4]
```diff
@@ -14,9 +14,9 @@ const OurWork = ({ descriptionText }) => {
       <Carousel>
         {CarouselSliderData.map((e, i, arr) => {
           return (
-            <div key={i} className="our-work__content">
-              <div className="our-work__content__description-container">
-                <div className="our-work__content__description-container__description">{descriptionText}</div>
+            <div key={i} className="work-content">
+              <div className="description-container">
```
Comment (imma-fish-u) [inline src/pages/home/sections/OurWork/index.js:18]: work-content__container

File: src/pages/home/sections/OurWork/index.js [hunk 5]
```diff
@@ -14,9 +14,9 @@ const OurWork = ({ descriptionText }) => {
...
+                <div className="description-container__description">{descriptionText}</div>
```
Comment (imma-fish-u) [inline src/pages/home/sections/OurWork/index.js:19]: work-content__description

File: src/pages/home/sections/Reviews/index.js [hunk 6]
```diff
@@ -13,24 +13,24 @@ const Reviews = ({ user, location, descriptionText }) => {
       <Carousel>
         {CarouselSliderData.map((e, i, arr) => {
           return (
-            <div key={i} className="reviews__content">
+            <div key={i} className="reviews-content">
```
Comment (imma-fish-u) [inline src/pages/home/sections/Reviews/index.js:16]: reviews__content

File: src/pages/home/sections/Reviews/index.js [hunk 7]
```diff
@@ -13,24 +13,24 @@ const Reviews = ({ user, location, descriptionText }) => {
...
               <img
                 src={e.image}
                 alt="our work"
                 width="650px"
                 height="513px"
               />
-              <div className="reviews__content__description">
+              <div className="description">
```
Comment (imma-fish-u) [inline src/pages/home/sections/Reviews/index.js:23]: reviews__description-wrapper

File: src/pages/home/sections/Reviews/index.js [hunk 8]
```diff
@@ -13,24 +13,24 @@ const Reviews = ({ user, location, descriptionText }) => {
...
                   <img src={formatQuote} alt="review" />
                   <CarouselItem
                     currentIndex={i + 1}
                     length={arr.length}
                   />
                 </div>
-                <div className="reviews__content__description__user">{user}</div>
-                <div className="reviews__content__description__location">{location}</div>
-                <div className="reviews__content__description__description">{descriptionText}</div>
+                <div className="description__user">{user}</div>
```
Comment (imma-fish-u) [inline src/pages/home/sections/Reviews/index.js:31]: reviews__user

File: src/pages/home/sections/Reviews/index.js [hunk 9]
```diff
@@ -13,24 +13,24 @@ const Reviews = ({ user, location, descriptionText }) => {
...
+                <div className="description__location">{location}</div>
```
Comment (imma-fish-u) [inline src/pages/home/sections/Reviews/index.js:32]: reviews__location

File: src/pages/home/sections/Reviews/index.js [hunk 10]
```diff
@@ -13,24 +13,24 @@ const Reviews = ({ user, location, descriptionText }) => {
...
+                <div className="description__description">{descriptionText}</div>
```
Comment (imma-fish-u) [inline src/pages/home/sections/Reviews/index.js:33]: reviews__description

**Review by imma-fish-u — CHANGES_REQUESTED** [review]

**Review by imma-fish-u — APPROVED** [review]

File: src/mockData/CarouselSliderData.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
+export const CarouselSliderData = [
+    {
+        image:
+            'https://images.unsplash.com/photo-1546768292-fb12f6c92568?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80'
```
Comment (EgorAntonovich) [inline src/mockData/CarouselSliderData.js:4]: Please think about way how we can optimise all consts in our FE proj!!!

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]

**Pull request closed without merging**